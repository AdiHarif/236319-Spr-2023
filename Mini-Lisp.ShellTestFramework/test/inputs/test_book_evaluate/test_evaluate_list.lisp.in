(evaluate-list NIL '())
(evaluate-list '(a) '(a.T))
(evaluate-list '(a b c) '(a.x b.y c.z))
