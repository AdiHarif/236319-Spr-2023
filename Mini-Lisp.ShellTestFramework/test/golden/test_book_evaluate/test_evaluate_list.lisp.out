NIL
(T)
(X Y Z)
Error [UNDEFINED] in UNDEFINED
